### PR TITLE
Bootstrap navbar

### PR DESCRIPTION
**Pull request by SULAIMAN-100**
Before sending your Pull Request, be sure to check that you are making the pull request from your branch to the dev branch! Please do not try to push directly into branches on the main repo.

### What changes have you made?

added bootstrap navbar

### CODE REVIEW

File: client/src/Pages/MentorsView.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ function MentorsView() {
   }
   const editLearningObjectives = (
     <a href="/MentorsEditLearningObj" className="signup-link">
-      Edit Learning Objectives
+      <button className="student-progress"> Edit Learning Objectives</button>
     </a>
```
Comment (petemcfarlane) [inline client/src/Pages/MentorsView.js]: It seems a bit odd to have a `<button>` inside an `<a>` tag, perhaps just choose one or the other? If you do decide to use a button, perhaps you could add a handler which changes the `window.location` `onClick`? Or if you decide to keep the `<a>` tag you could style it like a button.

File: client/src/components/Header.js [hunk 2]
```diff
@@ -3,31 +3,50 @@ import { BrowserRouter as Router, useLocation } from "react-router-dom";
...
-
-      <a href="/MentorsView">
-        <h3 className="signup-link">{back}</h3>
-      </a>
-      {logout}
-    </div>
+    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+      <div class="container-fluid">
+        <a class="nav-link" href={logoLink}>
+          {" "}
```
Comment (petemcfarlane) [inline client/src/components/Header.js]: Do you need the extra space here `{" "}` and after the `` ?

File: package.json [hunk 3]
```diff
@@ -66,7 +66,7 @@
     "@babel/runtime": "^7.10.5",
     "bcrypt": "^5.0.0",
     "body-parser": "^1.19.0",
-    "bootstrap": "^4.5.3",
+    "bootstrap": "^4.6.0",
```
Comment (petemcfarlane) [inline package.json:69]: Not sure if it makes a difference, but in "client/src/index.html" looks like you've upgraded bootstrap version from from 4.0.3 to 5.0.0-beta2, should we use that same version in the package.json? I don't know if it's available yet

**Review by petemcfarlane — APPROVED** [review]
What's the general reason behind this PR? Sorry I don't have the context. Is it to show a toggle button on smaller screens/devices?
Screenshot 2021-02-18 at 09 18 24

If so then it looks ok to me, just some minor feedback about buttons in anchor tags.

Feel free to add more context, Trello cards or screenshots to pull requests to make them even easier to review 👍

File: client/src/Pages/MentorsView.js (hunk 1 above)
Comment (AltomHussain) [reply client/src/Pages/MentorsView.js]: Got it thanks, the issue is been fixed

File: client/src/components/Header.js (hunk 2 above)
Comment (AltomHussain) [reply client/src/components/Header.js]: I think react added the extra space by default, we fixed it thanks.

File: package.json (hunk 3 above)
Comment (AltomHussain) [reply package.json:69]: I don't think we installed the bootstrap but we just used the CDN link to help implement the Navbar.

Comment (AltomHussain) [reply package.json:69]: We have fixed all the issues you mentioned in your feedback, except for the bootstrap one. We left it for a disscussion. 
Thanks @petemcfarlane

**Pull request merged**